### ISSUE TITLE
When the track is collapsed, observe how the title bars of some clips stick and some don't as you hover over them

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#7711]
When the track is collapsed, observe how the title bars of some clips stick and some don't as I hover over them. I assume the correct behavior here would be to show the title bar as long as the cursor hovers over the clip and to hide it as soon as it moves away. Am I right, @DilsonsPickles? Or do we need to show the title bar on each clip at all times?

https://github.com/user-attachments/assets/ece08881-32ea-4374-86d4-7fbadb4eb3c9

_Originally posted by @dozzzzer in https://github.com/audacity/audacity/issues/6768#issuecomment-2470426785_

**Comment by DilsonsPickles** [comment]
You are correct in your thinking. Title bar should only show on hover when clips are that small.

**Comment by Tantacrul** [comment]
@DilsonsPickles we should discuss this. If the header appears when you hover, what's the advantage of hiding it? It doesn't give the user any more space to make waveform selections right?

Also curious if there's a model you're basing this on. If so, it'd be good to try it. Otherwise, I'm wondering why it's needed?

Perhaps it's just not obvious from the bug here.

**Comment by DilsonsPickles** [comment]
The idea was simply to get the clip header out of the way to not obscure the waveform at smaller sizes. Everything you say is correct in terms of it not offering Users any additional space to make a selection. 

I don't have any _specific_ examples of this model being used on audio clips but the hover-on-something-to-show-additional-info is pretty commonplace.  

We might actually want to to do the reverse and show the header by default, and then move it out of the way on hover to afford the User more space for selection/editing. 

Let's discuss in design team meeting.

**Comment by Tantacrul** [comment]
OK, after a chat with @DilsonsPickles, we agreed to do the following:

- At small sizes, try reducing the height of the top bar.
- If the user starts editing wave data, then the header will disappear
- The user needs to deselect wave data in a clip for the header to be able to reappear again on hover

**Comment by DilsonsPickles** [comment]
Thanks, will provide new design link once it’s ready.

**Comment by chinakov** [comment]
Still reproducible on https://github.com/audacity/audacity/actions/runs/13849065281, latest master.

**Comment by chinakov** [comment]
Still reproducible on the latest master: be8bebce7a91042fedef5b0e5a98f9822d4bc0af

**Comment by chinakov** [comment]
Still reproducible on the latest Master (5983c6398fb9ebe208f254ce366d37c0938207de)

**Comment by chinakov** [comment]
https://github.com/user-attachments/assets/c5aedb39-cae6-42f6-a9ee-ae43878c83e0

The underlying issue has been fixed, and as far as I understand @DilsonsPickles changes will be addressed in a separate task?

**Comment by DilsonsPickles** [comment]
@chinakov - the video above is showing the intended behaviour, so I think you're safe to close this one :)
